xmtp/README.md

```diff
@@ -30,10 +42,17 @@ Installation:
 Message:
     - UNINITIALIZED: The message has not been encrypted yet
     - LOCALLY_COMMITTED: The outbound payloads have been constructed
+    - RECEIVED: The message is inbound and was retrieved from the network
 
 Outbound Payload:
     - PENDING: The payload has not been confirmed as sent yet
     - SERVER_ACKNOWLEDGED: The payload has been acknowledged by the server
+
+Inbound invite:
+    - PENDING: The payload has been downloaded from the network but has not been processed yet
+    - PROCESSED: The invite has been successfully processed and the conversation has been created
+    - DECRYPTION_FAILURE: The inner invite failed to decrypt
+    - INVALID: The invite failed validation
```

Comment:
For clarity this state the message decrypted successfully, however the inner packet could not be decoded?